PL_2_komentarze.py

```diff
@@ -0,0 +1,2722 @@
+        for i in obiektywarunkowklienta:
+            i.delete(0, END)
+            try:
+                i.validate()
+            except AttributeError:
+                pass
+        czypodcisn.set(0)
+        podcisn.disable()
+        tu.delete(0, END)
+        mediumBadania.current([0])
+        numerZam.delete(0, END)
+        tekstindeksuweza.set('')
+        czyzuzycie.set(0)
+        czyodtlu.set(0)
+        czyciaglosc.set(0)
+        poleuwagi.delete('0.0', END)
+        poleoznaczenia.delete('0.0', END)
+        typoznaczenia.current(0)
+        szukajweza.delete(0, END)
+        rezultaty.delete(0, END)
+        tekstindeksuweza.set('')
+        koncowki.current(0)
+        tuleje.current(0)
+        tulejwybor.current(0)
+    else:
+        pass
+
+def generatortolerancji():
+    flaga = listaparametrow[13]
+    dlu = int(dlugosc.get())
+    global toler
+    if flaga == 'P1':
+        if dlu <= 630:
+            toler = '-3 mm / +7 mm'
+        elif 630 < dlu <= 1250:
+            toler = '-4 mm / +12 mm'
+        elif 1250 < dlu <= 2500:
+            toler = '-6 mm / +20 mm'
+        elif 2500 < dlu <= 8000:
+            toler = '-0,5% / +1,5%'
+        elif 8000 < dlu:
+            toler = '-1% / +3%'
+    elif flaga == 'P2':
+        if dlu <= 630:
+            toler = '-4 mm / +12 mm'
+        elif 630 < dlu <= 1250:
+            toler = '-6 mm / +20 mm'
+        elif 1250 < dlu <= 2500:
+            toler = '-6 mm / +25 mm'
+        elif 2500 < dlu <= 8000:
+            toler = '-0,5% / +1,5%'
+        elif 8000 < dlu:
+            toler = '-1% / +3%'
+    elif flaga == 'P3':
+        if dlu <= 2500:
+            toler = '-6 mm / +25 mm'
+        elif 2500 < dlu <= 8000:
+            toler = '-0,5% / +1,5%'
+        elif 8000 < dlu:
+            toler = '-1% / +3%'
+    elif flaga == 'S1':
+        if dlu <= 1000:
+            toler = '+10 mm'
+        elif dlu > 1000:
+            toler = '+1%'
+    elif flaga == 'S2':
+        if dlu <= 1000:
+            toler = '+15 mm'
+        elif dlu > 1000:
+            toler = '+1,5%'
+    elif flaga == 'S3':
+        if dlu <= 1000:
+            toler = '+20 mm'
+        elif dlu > 1000:
+            toler = '+2%'
+    elif flaga == 'S4':
+        if dlu <= 1000:
+            toler = '+30 mm'
+        elif dlu > 1000:
+            toler = '+3%'
+    elif flaga == 'S5':
+        if dlu <= 1000:
+            toler = '+40 mm'
+        elif dlu > 1000:
+            toler = '+4%'
+    elif flaga == 'K':
+        if dlu <= 2500:
+            toler = '± 50 mm'
+        elif dlu > 2500:
+            toler = '± 2%'
+    elif flaga == 'C':
+        if dlu <= 1000:
+            toler = '+5%'
+        elif dlu > 1000:
+            toler = '+10%'
+    return toler
+
+
+class Tooltip:
+    '''
+    It creates a tooltips for a given widget as the mouse goes on it.
```

Comment:
Duży plus za to że zostawiasz takie informacyjne komentarze jak wklejasz kod z zewnątrz. W razie czego łatwo potem trafić w miejsce jego pochodzenia, doczytać albo dopytać o cos.